cloud_registry/config.yaml

```diff
@@ -53,9 +53,9 @@ db:
 api:
     specs:
         - path:
-            - api/20201108.11d2c12.service-info.yaml
-            - api/20201108.e0358db.openapi.yaml
-            - api/additions.openapi.yaml
+            - cloud_registry/api/20201108.11d2c12.service-info.yaml
+            - cloud_registry/api/20201108.e0358db.openapi.yaml
+            - cloud_registry/api/additions.openapi.yaml
```

Comment:
But this will only work when starting the app from the repository root directory, right? Would be good to implement this in a way that it can be started from anywhere, but this will probably require a fix in FOCA. I will take care of it during FOCA refactoring.